src/index.ts

```diff
@@ -41,6 +44,10 @@ export default class GDCPlugin extends Plugin {
       pluginManager.lib['@jbrowse/core/pluggableElementTypes/TrackType']
     const WidgetType =
       pluginManager.lib['@jbrowse/core/pluggableElementTypes/WidgetType']
+    // const InternetAccountType =
+    //   pluginManager.lib[
+    //     '@jbrowse/core/pluggableElementTypes/InternetAccountType'
```

Comment:
since the core work isn't merged into a release yet this can't work for now, uncomment and test once core work is merged in and these resources are available